js/angular-crmMailing/directives.js

```diff
@@ -28,6 +28,7 @@
           });
           scope.crmMailingConst = CRM.crmMailing;
           scope.ts = CRM.ts(null);
+          scope.helpFile = 'CRM/Mailing/Form/Upload';
```

Comment:
I assume everything in templates/CRM/Mailing will get deleted (we were targeting 4.7 for this, so could be quite soon now that we've branched).
So that leaves the 4th suggestion.